lib/plugins/aws/provider/awsProvider.test.js

```diff
@@ -344,19 +347,60 @@ describe('AwsProvider', () => {
     it('should get credentials from environment declared stage-specific profile', () => {
       const prevVal = process.env.AWS_TESTSTAGE_PROFILE;
       process.env.AWS_TESTSTAGE_PROFILE = 'notDefault';
-      const credentials = newAwsProvider.getCredentials('teststage', 'testregion');
+      const credentials = newAwsProvider.getCredentials();
       process.env.AWS_TESTSTAGE_PROFILE = prevVal;
       expect(credentials.credentials.profile).to.equal('notDefault');
     });
   });
 
-  describe('#getServerlessDeploymentBucketName', () => {
-    it('should return the name of the serverless deployment bucket', () => {
-      const options = {
-        stage: 'dev',
-        region: 'us-east-1',
+  describe('#getRegion', () => {
+    let newAwsProvider;
+
+    it('should prefer options over config or provider', () => {
+      const newOptions = {
+        region: 'optionsRegion',
+      };
+      const config = {
+        region: 'configRegion',
       };
+      serverless = new Serverless(config);
+      serverless.service.provider.region = 'providerRegion';
+      newAwsProvider = new AwsProvider(serverless, newOptions);
 
+      expect(newAwsProvider.getRegion()).to.equal(newOptions.region);
+    });
+    it('should prefer config over provider in lieu of options', () => {
```

Comment:
Please add a space above so that all tests follow the same pattern (should be applied to all the other tests below as well).